### DISCUSSION TITLE
HAA Security System controlling real Alarm System with Sonoff SV

### DISCUSSION THREAD

**Discussion by claire078** [RavenSystem/esp-homekit-devices#1872] [MEPLHAA Scripts]
Hi,
I used to use an SV with my Alarm System to control it:
{"c":{"l":13,"b":[{"g":0,"t":5}]},"a":[{"0":{"r":[{"g":12,"v":1,"i":1}]},"1":{"r":[{"g":12,"v":1,"i":1}]},"t":4,"b":[],"s":5},{"t":4,"b":[],"s":5,"f0":[{"g":4,"t":1}],"f1":[{"g":4,"t":0}]},{"t":12,"s":0,"f0":[{"g":14,"t":1}],"f1":[{"g":14,"t":0}]}]}

This SV was on old HAA boot, so I did an update to newest firmware.
HAA already has Security System, so I would like to use it.

I write it how would it work:

relay switches between OFF/Away, armed/disarmed to my Alarm System,
G4 input triggers armed (it not 100% necessary),
G14 input triggers alarm
Could someone create me the correct json code?
Thanks!

**Comment by dibehome** [comment]
Were you able to find the json? I am also interested.

**Comment by CryptonWind** [comment]
This is an example of Security System prepared for Sonoff SV which includes 2 motion sensors and one contact sensor:
The script also includes few more switches:
 - one switch for the relay of the real Siren
 - one switch for a virtual Siren - that I am using for Homekit scripts to trigger the alarm when Home and do some other staff, but not start the real Siren
 - one switch to trigger the real Siren when alarm activated
 - one switch to trigger the virtual Siren when alarm activated

The triggers are used to trigger the real/virtual siren from other Homekit sensors (not the above 3). 
The above mentioned sensors are triggering automatically the Alarm/Siren based on the internal service notifications.

{
  "c":{
    "b":[{"g":0,"t":5}],
    "l":13,
    "ntp":"ro.pool.ntp.org",
    "tz":"EET-2EEST,M3.5.0/3,M10.5.0/4"
  },
  "a":[{
    "3":{"m":[[3,0],[8,0]]},
    "4":{"m":[[3,1]]},
    "5":{"m":[[8,1]]},
    "6":{"m":[[3,1]]},
    "7":{"m":[[3,1]]},
    "t":55,
    "s":5,
    "es":[
      {
        "1":{"m":[[1,4]]},"i":1},
      {
        "0":{"r":[{"g":12}]},
        "1":{"r":[{"g":12,"v":1}]},
        "t":1,"b":[{"g":0}],"s":5,"i":3600},
      {
        "1":{"m":[[1,5],[1,6],[1,7]]},
        "t":12,"s":0,"i":180,
              "f1":[{"g":14,"t":1,"i":1}]},
      {
        "1":{"m":[[1,6]]},
        "t":12,"s":0,"i":180,
              "f1":[{"g":4,"t":1,"i":1}]},
      {
        "0":{"r":[]},"1":{"m":[[1,5],[1,6],[1,7]]},
        "t":5,"s":0,
              "f1":[{"g":5,"t":1,"i":0}],
              "f0":[{"g":5,"t":0}]},
      {
        "1":{"m":[[1,5]]},"i":1},
      {
        "t":1,"b":[],"s":0,"i":3600},
      {
        "t": 95,
        "h": [ 1, 0, 3 ]},
      {
        "t": 95,
        "h": [ 4, 0, 3 ]},
      {
        "t": 95,
        "h": [ 5, 0, 3 ]},
      {
        "t": 95,
        "h": [ 6, 0, 3 ]}
        ]}]}
